### DISCUSSION TITLE
There are garbled codes in the output Chinese

### DISCUSSION THREAD

**Discussion by TZ0385** [projectdiscovery/httpx#781] [Q&A] [answered]
![1](https://user-images.githubusercontent.com/44598912/191640365-3c8bf4fa-9d93-497f-bec8-76142bf3a598.png)
Windows 10 platform binary file tool, there are garbled codes on both sides of the output result

**Comment by forgedhallpass** [comment, marked as answer]
Those are ANSI color codes, that are not supported in Windows by default (at least without some registry modifications). You can either try using cywgin, git bash, or simply disable the colors with the `-nc` flag.

**Reply by TZ0385** [reply]
Thank you, solved
